resources/css/style.css

```diff
@@ -19,7 +19,7 @@ TopWidget {
 TopWidget::separator {
     background: none;
     border: none;
-    icon: none;
```

Comment:
Why remove this ?

Reply:
there is this warning
![image](https://user-images.githubusercontent.com/47149116/59349878-017b4500-8d1b-11e9-9fb2-907e5ae7f4fd.png)

Reply:
Ok, remove it instead of commenting it and we are good with this PR

Reply:
Done